src/main/java/ama/account/UserService.java

```diff
@@ -24,7 +24,7 @@ public User registerNewUserAccount(User user){
     }
 
     @Override
-    public User loadUserByUsername(String username) throws UsernameNotFoundException {
+    public User loadUserByUsername(String username) throws NullPointerException {
```

Comment:
I believe it was changed because userRepository.findByUsername() never actually throws a UsernameNotFoundException it simply returns null.